dart/test/sentry_client_test.dart

```diff
@@ -151,9 +170,10 @@ void main() {
       final client = SentryClient(options);
       await client.captureException(error, stackTrace: stackTrace);
 
-      final capturedEvent = (options.transport as MockTransport).events.first;
+      final capturedEnvelope =
+          (options.transport as MockTransport).envelopes.first;
+      final capturedEvent = await eventFromEnvelope(capturedEnvelope);
 
-      expect(capturedEvent.throwable, error);
       expect(capturedEvent.exception is SentryException, true);
       expect(capturedEvent.exception!.stackTrace, isNotNull);
```

Comment:
maybe we assert the error values then, not only if `stackTrace` is null, but also 1st frame file name, etc like we do in the other test `SentryClient captures exception and stacktrace`